apps/workflowengine/src/components/Checks/file.js

```diff
@@ -31,6 +31,19 @@ const FileChecks = [
 		validate: stringValidator,
 	},
 
+	{
+		class: 'OCA\\WorkflowEngine\\Check\\Directory',
+		name: t('workflowengine', 'Directory'),
+		operators: stringOrRegexOperators,
+		placeholder: (check) => {
+			if (check.operator === 'matches' || check.operator === '!matches') {
+				return '/^myfolder/.+$/i'
```

Comment:
This is an invalid placeholder.


```suggestion
				return '/^myfolder\/.+$/i'
```